src/simmate/workflows/README.md

```diff
@@ -666,6 +683,19 @@ The endpoint image supercell to use. This is really just a `structure` parameter
 The starting image supercell to use. This is really just a `structure` parameter under a different name, so everything about the `structure` parameter also applies here.
 
 
+## symmetry_precision
+If standardize_structure=True, then this is the cutoff value used to determine
+if the sites are symmetrically equivalent. (in Angstroms)
+``` python
+# python example
+symmetry_precision = 0.1
+```
+``` yaml
+# yaml file example
+symmetry_precision: 0.1
```

Comment:
@jacksund Is there a reason that this is 0.1 here but 0.01 above?  Similar to my other comment, 0.1 is a massive tolerance for a well-relaxed structure.  (I'm envisioning that this is only used after relaxation, not before.)  I ran a series of checks on my structures and I see the expected symmetrization at tolerances > 1e-4.  1e-3 would be sufficient in most cases and 1e-2 is sufficient in virtually every case.  1e-1 is likely to produce errors.

Reply:
Are you only looking at the final structure (after the final relaxation)? I found in my testing, 0.1A was needed at the 2nd relaxation stage in order to actually identify symmetries. Also what composition are you testing with? That could be another difference in why we're arriving at something different. 

Also, the `standardize_structure` parameter takes effect on the input structure btw -- changing the output risks making the results inaccurate or even creating mismatches in the site ordering/counts. I use 0.1A to **start** the 3rd relaxation and beyond.

Reply:
I'm doing this only after the final stage in the relaxation, testing with a Becca-type composition M<sub>7</sub>S<sub>8</sub>.

I hadn't realized that `standardize_structure` affected only input structures, but your statements about the risks make sense given this information.  So, once sub-workflows are operational (with a conditional relax & static), this issue disappears.

Reply:
> Becca-type composition M7S8

:laughing:

Reply:
Is it okay if I leave it at 0.1A for this PR? My testing lead me to 0.1 over 0.01 for early-on relaxations, so I want to do some more testing before I switch it back

Reply:
ok

Reply:
I'll respond back over in the main thread. But btw, the default is in fact 0.01. The code you highlighted here is just a documentation example of how to update the setting.